boards/xtensa/esp32/esp32-core/Kconfig

```diff
@@ -38,4 +38,10 @@ config ESP32CORE_RUN_IRAM
 		allows interoperability with the esp-idf system but makes you
 		reliant on the esp-idf design for these parts. Both are possible.
 
+config ESP32CORE_BINARY
+	bool "esp32 binary format"
```

Comment:
this is misleading as it's just a flash image.
how about "Flash image (mainly for QEMU)"

Reply:
Done.